tutorial/01-nuclide-naming.ipynb

```diff
@@ -14,8 +14,8 @@
      "source": [
       "Nuclide Naming Conventions\n",
       "==========================\n",
-      "One of the most basic aspects of nuclear software is how to uniquely respresent \n",
-      "nuclide names.  There exists the large number of different ways that people choose \n",
+      "One of the most basic problems in developing nuclear software is coming up with unique representations of \n",
```

Comment:
I don't think of this as a 'problem' per se.  And 'coming up with' is too informal.